src/Symfony/Bundle/FrameworkBundle/Templating/Helper/AssetsHelper.php

```diff
@@ -78,7 +78,7 @@ public function getVersion($path = null, $packageName = null)
 
         // packageName is null and path not, so path is a path or a packageName
         try {
-            $package = $this->packages->getPackage($path);
+            $this->packages->getPackage($path);
```

Comment:
Looks like just a dead code: 

the call is wrapped with try-catch for cases if the requested package load failed (BTW $e for exceptions name is really baaaaad - people should specify something like $pakageLoadFailed).

if loaded, the info taken from `$this->packages`.